Formula/bat.rb

```diff
@@ -19,8 +19,14 @@ class Bat < Formula
   def install
     ENV["SHELL_COMPLETIONS_DIR"] = buildpath
     system "cargo", "install", "--locked", "--root", prefix, "--path", "."
-    man1.install "doc/bat.1"
-    fish_completion.install "assets/completions/bat.fish"
+
+    # In https://github.com/sharkdp/bat/pull/673,
+    # documentation and fish autocompletion got parameterized
+    inreplace "assets/manual/bat.1.in", "{{PROJECT_EXECUTABLE | upcase}}", "bat"
+    inreplace "assets/manual/bat.1.in", "{{PROJECT_EXECUTABLE}}", "bat"
+    inreplace "assets/completions/bat.fish.in", "{{PROJECT_EXECUTABLE}}", "bat"
+    man1.install "assets/manual/bat.1.in" => "bat.1"
+    fish_completion.install "assets/completions/bat.fish.in" => "bat.fish"
```

Comment:
When `bat` is built, we actually generate a `bat.1` man page and a `bat.fish` completion file from these templates. They should land in the cargo output directory. There is no need to do this manually in the formula here.

See `build.rs` in `bat`:
https://github.com/sharkdp/bat/blob/136a745c5de804ee9cf0ec601302e17cef6334a3/build.rs#L35-L60

@eth-p FYI

Reply:
👋 @sharkdp, you are right, the compiled man and fish script do land it in here, https://github.com/sharkdp/bat/releases/download/v0.13.0/bat-v0.13.0-x86_64-apple-darwin.tar.gz

However, homebrew is building from the source code, which is the problem in here.

Reply:
> which is the problem in here.

No.. *as you are building from source*, these files are being generated by the custom `build.rs` script. Homebrew certainly runs `cargo build` at some point. This is when `bat.1` and `bat.fish` are being generated.

Reply:
That sounds about right, maybe we should change the flow to accommodate it. I did try to switch to use `cargo build` though, but get lost in the environment setup and the artifact output layout. 

Do you mind submitting a new PR to make the flow better?

Reply:
The files are also generated by `cargo install`. They only need to be `man1.install`ed/`fish_completion.install`ed from the right place. I'm not familiar with Homebrew, unfortunately. I don't have a MacOS system.

Reply:
I think I can see how to adjust the formula to do this a bit better. I’ll open a new PR!